src/pages/MyCakesList.css

```diff
@@ -0,0 +1,62 @@
+.My-cakes-page {
+  text-align: center;
+  display: grid;
+  height: 100vh;
+  grid-template-rows: 2rem 1fr 2rem;
+}
```

Comment:
I saw you did this on your Favs page too. It could be something you could have in the parent component, which I think in your case is Main. :)